src/platform/macos_iokit/enumeration.rs

```diff
@@ -128,8 +140,8 @@ pub(crate) fn probe_device(device: IoService) -> Option<DeviceInfo> {
         max_packet_size_0: get_integer_property(&device, "bMaxPacketSize0")? as u8,
         speed: get_integer_property(&device, "Device Speed").and_then(map_speed),
         manufacturer_string: get_string_property(&device, "USB Vendor Name"),
-        product_string: get_string_property(&device, "USB Product Name"),
-        serial_number: get_string_property(&device, "USB Serial Number"),
+        product_string,
+        serial_number,
```

Comment:
Using `.or_else()` would be a more concise way to write this.

```suggestion
        product_string: get_string_property(&device, "kUSBProductString")
            .or_else(|| get_string_property(&device, "USB Product Name")),
        serial_number: get_string_property(&device, "kUSBSerialNumberString")
            .or_else(||  get_string_property(&device, "USB Serial Number")),
```